### DISCUSSION TITLE
Pop! OS 22.04 Locked Screen

### DISCUSSION THREAD

**Discussion by hkdb** [rustdesk/rustdesk#1106] [General]
When the screen is locked and blanked for more than 5-10 seconds, I seem to loose keyboard input after the screen is waken with a mouse click or keyboard stroke from a Rustdesk remote session. Then, the keyboard input is completely unresponsive. The only way out of it is if I locally do ctrl-alt-f1 (so I guess I don't loose all keyboard input after all) and login. Then it will bring me back to my desktop unlocked and intact.

I am able to reproduce this on 2 machines running Pop! OS 22.04. My config is wayland disabled for gdm and gnome is running on xorg. I also have the "Allow Locked Remote Desktop" gnome-shell extension installed.

This seems to happen too if I try to use VNC, RDP, or even Barrier (not 100% of the time for barrier) with varying results but the general premise of keyboard input is unresponsive in remote sessions persists. Maybe keyboard input is blocked by something? Is there a work around?

Any thoughts?

Thanks!

**Comment by rustdesk** [comment]
I have no work around either.

**Comment by hkdb** [comment]
Actually, I think it's a resolved bug in gnome-shell. A couple days ago, I noticed a bunch of new updates coming in. gnome-shell and remote-desktop were included. After the update, the problem was resolved. Thanks!

**Reply by rustdesk** [reply]
Great.
